lib/ClangImporter/ImporterImpl.h

```diff
@@ -1560,6 +1560,9 @@ bool isSpecialUIKitStructZeroProperty(const clang::NamedDecl *decl);
 /// even if imported as a non-static member function.
 bool isImportedAsStatic(clang::OverloadedOperatorKind op);
 
+bool hasSameUnderlyingType(const clang::Type *a,
```

Comment:
A small comment about what this does would be nice as it's not immediately obvious from the name.